eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -144,10 +152,45 @@ def write_output(self, dir_out):
             for evidence_string in self.evidence_string_list:
                 fdw.write(json.dumps(evidence_string) + '\n')
 
-        with utilities.open_file(dir_out + '/' + config.EVIDENCE_RECORDS_FILE_NAME, 'wt') as fdw:
-            for evidence_record in self.evidence_list:
-                evidence_record_to_output = ['.' if ele is None else ele for ele in evidence_record]
-                fdw.write('\t'.join(evidence_record_to_output) + '\n')
+        with utilities.open_file(os.path.join(dir_out, config.ZOOMA_FILE_NAME), "wt") as zooma_fh:
+            zooma_fh.write("STUDY\tBIOENTITY\tPROPERTY_TYPE\tPROPERTY_VALUE\tSEMANTIC_TAG\tANNOTATOR\tANNOTATION_DATE\n")
+            date = strftime("%d/%m/%y %H:%M", gmtime())
+            with utilities.open_file(dir_out + '/' + config.EVIDENCE_RECORDS_FILE_NAME, 'wt') as fdw:
```

Comment:
Can this with-block and the for-loop below be extracted to their own methods and documented to stress what the difference between them is?